release.yml

```diff
@@ -0,0 +1,123 @@
+# JIRA Project code where the release issue should be created
+jiraProject:     SB
+
+# If already created, provide the issue number of the Release, otherwise keep it empty.
+# It will be used by the command create-component to link the stakeholder issues
+jiraKey:         # SB-1484
+
+# Version to be released. This version will be used to populate the description to be created for each Jira stakeholder: component or starter owner
+version:         2.3.0.RELEASE
+
+# Long version name used to create the title of the release issue
+longVersionName: "[Spring Boot 2.3.0] Release steps CR [Q3-2020]"
+
+# Release date scheduled. It will also be used to populate the description to be created for each Jira stakeholder: component or starter owner
+releaseDate:     September 14th 2020
+
+# Due date for the component/starter owner to release their new components, QE signoff it
+dueDate:         2020-08-21
+
+# EOL of the Snowdrop release
+eol:             September 2021
+
+components:
+  # List of JIRA Projects contributing to a Snowdrop release
+  # They will be used to create a component or starter
+  #
+  # https://issues.redhat.com/projects/KEYCLOAK/issues/
+  # https://issues.redhat.com/projects/JWS/issues
+  # https://issues.redhat.com/projects/JDG/issues
+  # https://issues.redhat.com/projects/RESTEASY/issues
+  # https://issues.redhat.com/projects/TRACING/issues
+  # https://issues.redhat.com/projects/EAPSUP/issues
+  # https://issues.redhat.com/projects/ENTMQCL/issues
+  #
+  - jiraProject: SB
+    jiraTitle: "Hibernate version to use for SB 2.3"
+    name: Hibernate
+    version: 5.4.15
```

Comment:
Where is that `version` field coming from and what does it correspond to?

Reply:
This is the version coming from the Spring Boot Dependencies bom released by Spring for 2.3.0 - https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.3.0.RELEASE/spring-boot-dependencies-2.3.0.RELEASE.pom

same remark too for components such as undertow, tomcat, ...

Remark: in fact this is suggested version